db/data_migrations/20160920123015_add_national_footnote.rb

```diff
@@ -0,0 +1,362 @@
+TradeTariffBackend::DataMigrator.migration do
+  name "Add national footnote to Taric's XXXXX"
+
+  FOOTNOTE_TYPE_ID = "XX"
+  FOOTNOTE_ID = "XXX"
+  FOOTNOTE_DESCRIPTION = "Where the free rate of duty has been applied due to inclusion of 'Multi-component integrated circuits' (MCOs) document code Y035 (no status or reference required) must be declared at item level in box 44."
+
+
+  TARIC_FOOTNOTE_TYPE_ID = "CD"
```

Comment:
yeah the old migration was adding footnotes where there was an existing taric migration, which is why this and code relating to this was here - this one we are just adding in a footnote directly to commodity codes.